GeosCore/cldj_interface_mod.F90

```diff
@@ -1213,4 +1216,141 @@ FUNCTION Get_RH_Index( RH ) RESULT( index )
+    ! CloudJ_DryRun_Check begins here!
+    !========================================================================
+    amIRoot = Input_Opt%amIRoot
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'CJ77_inp.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'FJX_spec.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'FJX_scat-aer.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'FJX_scat-cld.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'FJX_scat-ssa.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'FJX_scat-UMa.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'FJX_scat-geo.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'atmos_std.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'atmos_h2och4.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'atmos_geomip.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+    fileName = TRIM( Input_Opt%CloudJ_Dir ) // 'FJX_j2j.dat'
+    CALL Check_File_For_DryRun( fileName, amIRoot )
+
+  END SUBROUTINE CloudJ_DryRun_Check
+!EOC
+!------------------------------------------------------------------------------
+!                  GEOS-Chem Global Chemical Transport Model                  !
+!------------------------------------------------------------------------------
+!BOP
+!
+! !IROUTINE: Check_File_For_DryRun
+!
+! !DESCRIPTION: Checks the status of a single file in dry-run simulations.
+!\\
+!\\
+! !INTERFACE:
+!
+ SUBROUTINE Check_File_for_DryRun( fileName, amIRoot )
+!
+! USES:
+!
+    USE ErrCode_Mod
+    USE Input_Opt_Mod, ONLY : OptInput
+!
+! !INPUT PARAMETERS:
+!
+    CHARACTER(LEN=*), INTENT(IN) :: fileName   ! Name of file to check
+    LOGICAL,          INTENT(IN) :: amIRoot    ! Are we on the root core?
+!
+! !REVISION HISTORY:
+!  19 Dec 2023 - R. Yantosca - Initial version
+!  See https://github.com/geoschem/geos-chem for complete history
+!EOP
+!------------------------------------------------------------------------------
+!BOC
+    
+    ! Scalars
+    LOGICAL            :: fileExists
+
+    ! Strings
+    CHARACTER(LEN=512) :: fileMsg
+
+    !========================================================================
+    ! Check_File_for_DryRun begins here!
+    !========================================================================
+
+    ! Test if the file exists
+    INQUIRE( FILE=TRIM( fileName ), EXIST=fileExists )
+
+    ! Test if the file exists and define an output string
+    IF ( fileExists ) THEN
+       fileMsg = 'CLOUD-J: Opening'
```

Comment:
This message may be misleading since CLOUD-J is not opening the file. How about "DryRun: Opening file required for CLOUD-J"?

Reply:
@lizziel: I tried to match the existing syntax of dryrun output, which gives an idea of where the file is being read from.  I can change this to something like `DRYRUN (Cloud-J) ...` or whatever we think.
```console
HEMCO: OPTIONAL FILE NOT FOUND ./Restarts/HEMCO_restart.201907010000.nc
HEMCO: Opening ./Restarts/GEOSChem.Restart.20190701_0000z.nc4
CLOUD-J: Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/CLOUD_J/v2023-05/CJ77_inp.dat
CLOUD-J: Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/CLOUD_J/v2023-05/FJX_spec.dat
CLOUD-J: Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/CLOUD_J/v2023-05/FJX_scat-aer.dat
CLOUD-J: Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/CLOUD_J/v2023-05/FJX_scat-cld.dat
CLOUD-J: Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/CLOUD_J/v2023-05/FJX_scat-ssa.dat
CLOUD-J: Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/CLOUD_J/v2023-05/FJX_scat-UMa.dat
CLOUD-J: Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/CLOUD_J/v2023-05/FJX_scat-geo.dat
CLOUD-J: Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/CLOUD_J/v2023-05/atmos_std.dat
CLOUD-J: Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/CLOUD_J/v2023-05/atmos_h2och4.dat
CLOUD-J: Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/CLOUD_J/v2023-05/atmos_geomip.dat
CLOUD-J: Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/CLOUD_J/v2023-05/FJX_j2j.dat
CLOUD-J (RD_AOD): Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2021-10/so4.dat
CLOUD-J (RD_AOD): Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2021-10/soot.dat
CLOUD-J (RD_AOD): Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2021-10/org.dat
CLOUD-J (RD_AOD): Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2021-10/ssa.dat
CLOUD-J (RD_AOD): Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2021-10/ssc.dat
CLOUD-J (RD_AOD): Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2021-10/h2so4.dat
CLOUD-J (RD_AOD): Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2021-10/h2so4.dat
CLOUD-J (RD_AOD): Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2021-10/dust.dat
CLOUD-J (RD_PROF_NC): Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FastJ_201204/fastj.jv_atms_dat.nc
UCX (SFCMR_READ): Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/UCX_201403/NoonTime/Grid4x5/InitCFC_O_01.dat
UCX (SFCMR_READ): Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/UCX_201403/NoonTime/Grid4x5/InitCFC_O1D_01.dat
```

Reply:
See comment: https://github.com/geoschem/geos-chem/pull/2082#issuecomment-1879236534